src/civic/index.js

```diff
@@ -347,10 +381,10 @@ const processEvidenceRecord = async (opt) => {
     const content = {
         conditions: [...variants.map(v => rid(v))],
         description: rawRecord.description,
-        evidence: [rid(publication)],
+        evidence: [rid(publication, true)],
```

Comment:
The publication should never be null. If it is then something has gone wrong and we should throw an error rather than create the statement

Reply:
Yeah, we got some errors appening without this argument. I used it because it was already available on the function. I'll roll that back and we'll see exactly which case poses problem.